tests/integration/cloud_governance/common/zombie_cluster/test_ec2_zombie_delete.py

```diff
@@ -0,0 +1,46 @@
+import datetime
```

Comment:
Need to verify that we dont expose data when running it

Reply:
verified sir. No data exposed.